### ISSUE TITLE
helmfile test pod logs

### ISSUE THREAD

**Issue by sanjay-nag** [roboll/helmfile#1541]
How do I dump logs from the pods created through `helmfile test`?

I find `helm test` has `cleanup`, `logs`, and `parallel` flags. What are the equivalents in `helmfile test`?

**Comment by mumoshu** [comment]
@sanjay-nag Good catch. We should definitely implement `helmfile test --logs` to add the flag to `helm test` called by Hemlfile 😃 Do you think supports for `cleanup` and `parallel` would also be useful? Then I'd implement them all.

**Comment by sanjay-nag** [comment]
@mumoshu thank you. I see `cleanup` already implemented, my bad! `parallel` would be a great addition - I would definitely use it.

**Comment by mumoshu** [comment]
@sanjay-nag `--logs` will be added soon via #1569

Unfortunately, `--parallel` turned out to be not yet ported to helm 3 according to https://github.com/helm/helm/pull/4144#issuecomment-585128017.
The good news is that it's coming in https://github.com/helm/helm/pull/8946!
